hugr-py/tests/serialization/test_extension.py

```diff
@@ -98,7 +98,6 @@ def test_extension():
         version=SemanticVersion(0, 1, 0),
         name="ext",
         types={"foo": type_def},
-        values={},
```

Comment:
This change I don't understand - it seems out of place?

Reply:
yeah it's a drive by, values in extensions were also removed but this test wasn't updated